src/libs/actions/IOU.ts

```diff
@@ -6448,7 +6448,7 @@ function getReportFromHoldRequestsOnyxData(
         chatReport.reportID,
         chatReport.policyID ?? iouReport.policyID ?? '',
         recipient.accountID ?? 1,
-        holdTransactions.reduce((acc, transaction) => acc + transaction.amount, 0) * -1,
+        holdTransactions.reduce((acc, transaction) => acc + transaction.amount, 0) * (ReportUtils.isExpenseReport(iouReport) ? -1 : 1),
```

Comment:
shouldn't you specifically check for IOU reports for 1? Because what about invoiceReports, smartReports ... etc...?

Reply:
oh, good call 🤔 I'll update